packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -61,221 +45,64 @@ const propTypes = {
    */
   postMenstrualAge: PropTypes.string,
   /**
-   * Label to display for the PostMenstrualAgeLabel
-   */
-  postMenstrualAgeLabel: PropTypes.string,
-  /**
    * The persons preferred first name if they have one.
    */
   preferredFirstName: PropTypes.string,
 };
 
 const defaultProps = {
   applicationContent: null,
-  age: '--',
-  dateOfBirth: '--',
-  dateOfBirthLabel: 'DOB',
+  age: undefined,
+  dateOfBirth: undefined,
   deceasedDate: null,
-  deceasedDateLabel: 'Deceased',
-  gender: '--',
+  gender: undefined,
   gestationalAge: null,
-  gestationalAgeLabel: 'GA',
   identifiers: {},
-  personName: '--',
+  personName: undefined,
   photo: null,
   postMenstrualAge: null,
-  postMenstrualAgeLabel: 'PMA',
   preferredFirstName: null,
 };
 
-// eslint-disable-next-line react/prop-types
-const DemographicsBannerValue = ({ label, value }) => (
-  <span className="terra-DemographicsBanner-value">
-    { label && <span className="terra-DemographicsBanner-value-label">{`${label}:`}</span> }
-    <b>{value}</b>
-  </span>
-);
-
-class DemographicsBanner extends React.Component {
-  personDetails() {
-    const elements = [
-      <DemographicsBannerValue key="age" value={this.props.age} />,
-      <DemographicsBannerValue key="gender" value={this.props.gender} />,
-      <DemographicsBannerValue
-        key="dob"
-        label={this.props.dateOfBirthLabel}
-        value={this.props.dateOfBirth}
-      />,
-    ];
-
-    if (this.props.gestationalAge) {
-      elements.push(
-        <DemographicsBannerValue
-          key="ga"
-          label={this.props.gestationalAgeLabel}
-          value={this.props.gestationalAge}
-        />,
-      );
-    }
-
-    if (this.props.postMenstrualAge) {
-      elements.push(
-        <DemographicsBannerValue
-          key="pma"
-          label={this.props.postMenstrualAgeLabel}
-          value={this.props.postMenstrualAge}
-        />,
-      );
-    }
-
-    if (this.props.deceasedDate) {
-      elements.push(
-        <DemographicsBannerValue
-          key="deceased"
-          label={this.props.deceasedDateLabel}
-          value={this.props.deceasedDate}
-        />,
-      );
+const contextTypes = {
```

Comment:
The designed behavior is: if the DemoBanner isn't inside the Base with locale(which provides context.intl), it needs to throw the friendly error message to tell consumers to do that. The reason behind that is, as we support the consumer, who doesn't want to have translations, to use Base without locale, we need to throw such error message if they try to use translated component(like DemoBanner) inside untranslated Base.